src/MemberProfile.js

```diff
@@ -1,17 +1,115 @@
+import FontIcon from 'material-ui/FontIcon';
+import AppBar from 'material-ui/AppBar';
+import SvgIconFace from 'material-ui/svg-icons/action/face';
 
 const MemberProfile = ({ user }) => {
   return (
-    <div>
-      <img src={user.picture} alt="avatar" />
-      <h1>{user.name}</h1>
-      <p>{user.description}</p>
-      <p>{user.email}</p>
-      <p>GitHub: {user.githubName}</p>
-      <p>Flowdock: {user.flowdockName}</p>
-      <Link to="/me/edit">edit</Link>
+  <MuiThemeProvider>
+    <div className="container">
+      <div className="card">
+        <div className="pic">
+          <div className="container-nested">
+            <img src="http://placehold.it/225x225"/>
+          </div>
+        </div>
+        <div className="user-bio">
+          <div className="container-nested">
+            <h2>FirstName LastName</h2>
+            <p>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut 
+            labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+            nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit 
+            esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in 
+            culpa qui officia deserunt mollit anim id est laborum."</p>
+            <a href="mailto:email@gmail.com">email@gmail.com</a>
+              <div className= "contact">
+                <span>GitHub: </span><a href="github.com">github.com</a>
+              </div>
+              <div className= "contact">
+                <span>Flowdock: </span><a href= "flowdock.com">flowdock.com</a>
+              </div>
+            </div>
+          </div>
+    <div className="user-experience">
+      <div className="contributions-nested">
+        <div className= "contributed">
+          <div>
+            <span className="hours-contributed">
+              1
+            </span> 
+              <div className="details">
+              hours contributed
+              </div>
+          </div>
+        </div>
+        <div className= "contributed1">
+          <div>
+            <span className="projects-completed">
+                5
+            </span> 
+              <div className="details">
+              projects completed
+              </div>
+          </div>
+        </div>
+        <div className= "contributed2">
+          <div>
+            <span className="hours-contributed">
+                35
+            </span> 
+              <div className="details">
+              meetups attended
+              </div>
+          </div>
+        </div>
+      </div>
     </div>
+  </div>
+
+      <div className="card">
+        <div className="skillsleft">
+          <h2>Skills</h2>
+        </div>
+        <div className="skillsright"> 
+          <Chip className="skills-button" 
+          onRequestDelete= {()=> {return false}}>
+            Python
```

Comment:
Maybe we could put a pending indicator in here as well. Maybe we make it into a separate react component that is an ellipses (...) or something that when you mouse over it says: `Help us implement this feature`